composer.json

```
@@ -138,7 +138,7 @@
     },
     "scripts-descriptions": {
         "check-cs": "Run code style checker for all files",
-        "fix-cs": "Fix Coding standard issues in current checkout.",
+        "fix-cs": "php-cs-fixer fix --config=.php-cs-fixer.php --show-progress=dots",
```

Too much automation ;)
```suggestion
        "fix-cs": "Fix Coding standard issues in current checkout.",
```